android/src/com/mapswithme/maps/maplayer/SearchWheel.java

```diff
@@ -184,7 +175,7 @@ public void onResume()
     }
 
     final String query = SearchEngine.INSTANCE.getQuery();
-    if (TextUtils.isEmpty(query))
+    if (TextUtils.isEmpty(query) || query == null)
```

Comment:
TestUtils.isEmpty should check for null

Reply:
Same issue as with `mSearchLayout`, if we remove this check the IDE will show a warning.

Reply:
Well, I do prefer clean code instead of IDE _strange_ tips ;)